charts/zookeeper/templates/zookeeper.yaml

```diff
@@ -3,7 +3,15 @@ kind: "ZookeeperCluster"
 metadata:
   name: {{ template "zookeeper.fullname" . }}
 spec:
-  replicas: 3
+  replicas: {{ .Values.replicas }}
   image:
     repository: {{ with .Values.image }}{{ .repository }}{{ end }}
     tag: {{ with .Values.image }}{{ .tag }}{{ end }}
+    pullPolicy: {{ with .Values.image }}{{ .pullPolicy }}{{ end }}
+  persistence:
+    accessModes: [ "ReadWriteOnce" ]
+    storageClassName: {{ .Values.storage.className }}
+    reclaimPolicy: Delete
```

Comment:
Can we add reclaimPolicy also in values.yaml

Reply:
done